DOC-470 Experiment Overview rewrite (#526)
amplitude/amplitude-docs: content/collections/experiment/en/overview.md

  - 'Familiarize yourself with the concepts and processes involved in creating an experiment or feature flag in Amplitude Experiment'
landing: true
exclude_from_sitemap: false
updated_by: 13054dd3-3dcd-4d55-aaaf-13bb99388147
updated_at: 1740433491
sourxe: 'https://help.amplitude.com/hc/en-us/articles/360061270232-Amplitude-Experiment-overview-Optimize-your-product-experience-through-A-B-testing'
landing_blurb: 'Learn the value of experimentation in your product.'
---

> For decades, product teams have relied on **experimentation** as a way to rank and implement product adjustments. But it’s never been easy. Because of that, these experiments often just tweak peripheral issues around the margins, instead of driving the big-picture changes that optimize the product experience.

Reviewer: nit: One of our previous writers loved bolding words. Please remove them in instances like this paragraph.

Author: No problem. I kept the bolding just incase it was a style the company liked.

Amplitude Experiment is a workflow-driven behavioral experimentation platform that actually **accelerates** your roadmap, letting you focus on making better decisions and building better products.

With Experiment, you can easily **modify and configure product experiences** for unique audiences through:

> * **Product experimentation**: Improve key KPIs by **running experiments and A/B tests** to onboard new users, reduce friction for checkout experiences, roll out new features, and more.

Author: `**Product experimentation**`

@markzegarelli  Would we remove the bolding from here too?

{{partial:admonition type='note'}}
 If your organization has purchased the [Accounts add-on,](/docs/analytics/account-level-reporting) you may perform bucketing and analysis on groups rather than users.
{{/partial:admonition}}

## Allocate users

Decide how many users should see your variants by rolling the experiment out to the entire user base or just a fraction. Specify the number of users for the control and variant experiences, define user segments to include or exclude, and even target specific user or device IDs.

## Activate your experiment and analyze your results

> When your experiment is set up, you can select *Start Experiments* to send it to live users.

Reviewer: ```suggestion
When your experiment is set up,  click *Start Experiments* to send it to live users.
```

nit: "you can" is unnecessary places like this.

Also, this doesn't really send experiments to users, but rather enables the experiment so that users can be exposed to it.

{{partial:admonition type='note'}}
Behind the scenes, experiments and flags are very similar, but the basic difference is this: An experiment helps you make sure you’re building the right thing for your business, while feature flags allow seamless feature releases and rollbacks. This is because an experiment has metrics and a feature flag doesn't, giving you greater flexibility to mess with controls without breaking anything.
{{/partial:admonition}}

Once you’ve configured your deployment, go straight to creating your variants. The basic idea, a new and different product experience that some users see but others don't—remains the same. But instead of exploring how different user segments react to different user experiences, you’ll be choosing which users get access to new features first. When working with feature flags, the variant represents code for a new feature that isn’t yet released to your entire user base.

> You are allocating users to your variants as you would if you were running an experiment, and activating your flag is as simple as switching on your experiment.

Reviewer: ```suggestion
You're allocating users to your variants as you would if you were running an experiment, and activating your flag is as simple as switching on your experiment.
```

Please use contractions wherever possible
